assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/keycloak/oidcCallbackDashboard.html

```diff
@@ -0,0 +1,23 @@
+<!--
+
+    Copyright (c) 2015-2017 Red Hat, Inc.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+    Contributors:
+      Red Hat, Inc. - initial API and implementation
+
+-->
+<html>
+<head>
+  <title>oidcCallback</title>
+  <script type="text/javascript" src="./oidcCallback.js"></script>
+</head>
+<body>
+<script type="text/javascript">
+  redirectToInitialPage(false);
```

Comment:
Can we do this as `<body onload="redirectToInitialPage(false)"/>` ?

Reply:
Certainly possible. I'll look into this.